tests/metrics/test_LPIPS.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import torch
+from skimage import data
+from torchmetrics.image.lpip import LearnedPerceptualImagePatchSimilarity
+
+from meddlr.metrics.LPIPS import LPIPS
+
+
+def test_LPIPS_torchmetrics_reproducibility():
+    """Test reproducibility between SSIM implementations in meddlr and torchmetrics."""
+
+    meddlr_metric = LPIPS(mode="rgb", lpips=True)
+
+    preds = torch.randn((5, 3, 256, 256))
+    targets = torch.randn((5, 3, 256, 256))
+
+    targets = targets + 1j * targets
+    preds = preds + 1j * preds
+
+    meddlr_metric(preds, targets)
+    meddlr_out = meddlr_metric.compute().sum()
+
+    torchmetrics_metric = LearnedPerceptualImagePatchSimilarity(net_type="vgg", reduction="sum")
+    # torchmetrics lpips has no built-in preprocessing so we use our own
+    preds = LPIPS(mode="rgb").preprocess_lpips(preds)
+    targets = LPIPS(mode="rgb").preprocess_lpips(targets)
+
+    torchmetrics_out = torchmetrics_metric(preds, targets)
+
+    assert torch.allclose(meddlr_out, torchmetrics_out)
+
+
+def test_LPIPS_noise():
+    """Test that LPIPS increases as Gaussian noise increases"""
+
+    metric = LPIPS()
+    noise_levels = [0, 0.01, 0.05, 0.1, 0.25]
+
+    target = data.camera().astype(np.float32)
+    target = target + 1j * target
+    targets = np.repeat(target[np.newaxis, np.newaxis, :, :], len(noise_levels), axis=0)
```

Comment:
add comment with shape of this tensor, is it [5, 1, H, W]?

Reply:
yup